bundlewrap/items/dconf.py

```diff
@@ -107,7 +109,7 @@ def patch_attributes(self, attributes):
 
     @classmethod
     def validate_attributes(cls, bundle, item_id, attributes):
-        if not attributes.get('value') and not attributes.get('reset'):
+        if 'value' not in attributes and 'reset' not in attributes:
```

Comment:
setting `reset` to `False` should be handled here, `cdict()` also demands a true-ish value for it to work

Reply:
the docs should probably also mention what value `reset` should be set to